### PR TITLE
Make it possible to show QR code in address page

### PR DESCRIPTION
**Pull request by rantan**
And also make it possible to work production build

### CODE REVIEW

File: frontend/angular.json [hunk 1]
```diff
@@ -178,7 +178,8 @@
     }
   },
   "cli": {
-    "defaultCollection": "@ionic/angular-toolkit"
+    "defaultCollection": "@ionic/angular-toolkit",
+    "analytics": "32fc55d0-5812-45e2-aa87-9a88467b1b53"
```
Comment (Yamaguchi) [inline frontend/angular.json:182]: Is this line needed?

**Review by Yamaguchi — CHANGES_REQUESTED** [review]

File: frontend/angular.json (hunk 1 above)
Comment (rantan) [reply frontend/angular.json:182]: No. I will remove it.

File: frontend/src/app/address/address.page.html [hunk 2]
```diff
@@ -23,9 +23,12 @@
         </ion-col>
         <ion-col size="4" class="ion-text-end">
           <ngx-qrcode
-            [(qrc-value)]="qrcode"
-            qrc-element-type="url"
-          ></ngx-qrcode>
+            elementType="url"
+            [value]="qrcode"
+            cssClass="aclass"
```
Comment (Yamaguchi) [inline frontend/src/app/address/address.page.html:28]: "aclass" is not defined in css.

**Review by Yamaguchi — CHANGES_REQUESTED** [review]

File: frontend/src/app/address/address.page.html (hunk 2 above)
Comment (rantan) [reply frontend/src/app/address/address.page.html:28]: Thanks, I just pated example in the library document... I will remove it.

**Pull request merged**